controllers/drclusters.go

```diff
@@ -119,6 +119,12 @@ func objectsToDeploy(hubOperatorRamenConfig *rmn.RamenConfig) ([]interface{}, er
 		return nil, err
 	}
 
+	if drClusterOperatorRamenConfig.RamenOpsNamespace != "" {
```

Comment:
The ops NS would be deleted when the DRCluster manifest is deleted, while this deletion would happen once all DRPolicies that refer to this DRCluster are deleted and we no longer need the ops NS, the point that the operator and operands would be in different namespaces and a delete of one will not impact the delete of the other would be missed.

Overall this is not an issue, as deletion of the DRCluster should already ensure all DRPCs (and hence VRGs delivered to the ops NS) are clean, just noting the behavior in this comment.

Reply:
Ok, this would be bad. I filed #1393 to address this.